index.js

```diff
@@ -1,6 +1,11 @@
 import Twitter from 'twit';
+import Promise from 'pinkie-promise';
 
-export default function getTwitterInfo(tokens, username, cb) {
+export default function getTwitterInfo(tokens, username) {
   const client = new Twitter(tokens);
-  client.get('users/show', { screen_name: username }, cb);
+
+  return new Promise((resolve, reject) => {
+    const handler = (err, res) => { err ? reject(err) : resolve(res) };
+    client.get('users/show', { screen_name: username }, handler);
+  });
```

Comment:
just take a look how it looks now https://github.com/iamstarkov/get-twitter-info/blob/master/index.js